catalog/spatial/registry/registry-schema-bindings/src/main/java/org/codice/ddf/registry/schemabindings/RegistryPackageUtils.java

```diff
@@ -172,6 +253,23 @@ public static SlotType1 getSlotByName(String name, List<SlotType1> slots) {
         return slotAttributes;
     }
 
+    public static Map<String, List<SlotType1>> getSlotMapWithMultipleValues(List<SlotType1> slots) {
```

Comment:
The registryPackageUtils just handles parsing and returning RegistryObjectTypes. If this is needed in a helper it should probably be added to the RegistryPackageWebConverter.